src/runtime_src/core/tools/common/tests/TestGemm.cpp

```diff
@@ -196,7 +197,7 @@ TestGemm::run(std::shared_ptr<xrt_core::device> dev)
   std::this_thread::sleep_for(std::chrono::milliseconds(30));
 
   //Calculate TOPS
-  double ipu_hclck_period= 1000000000.0/(ipu_hclock*1000000); //MHz to ns
+  double ipu_hclck_period= 1000000000.0/((ipu_hclock != 0 ? ipu_hclock : 1)*1000000); //MHz to ns
```

Comment:
If `(ipu_hclock == 0) `should we fail the test?

Reply:
Yes, I've made a change to do so